docs/guide_restli_client.md

```diff
@@ -583,25 +625,28 @@ appropriate for the corresponding resource method:
 -   `input(V entity)` - sets the input payload for the request
 -   `inputs(Map<K, V> entities)` - sets the input payloads for batch
     requests
+-   `returnEntity(boolean value)` - sets the [`$returnEntity` query parameter](/rest.li/spec/return_entity#query-parameter)
 
 The following table summarizes the methods supported by each
 RequestBuilder type.
 
-| Request Builder | header | id | ids | name | setParam | addParam | assocKey | pathKey | paginate | fields | input | inputs |
-| Action | - | - | | - | - | - | | - | | | | |
-| Find | - | | | - | - | - | - | - | - | - | | |
-| Get | - | -\* | | |- | - | | - | | - | | |
-| Create | - | | | | - | - | | - | | | - | |
-| Delete | - | -\* | | |- | - | | - | | | | |
-| PartialUpdate | - | - | | | - | - | | - | | | - | |
-| Update | - | -\* | | |- | - | | - | | | - | |
-| BatchGet | - | | - | | - | - | | - | | - | | |
-| BatchCreate | - | | | | - | - | | - | | | | - |
-| BatchDelete | - | | - | | - | - | | - | | | | |
-| BatchPartialUpdate | - | | | | - | - | | - | | | | - |
-| BatchUpdate | - | | | | - | - | | - | | | | - |
-
-*It is not supported, if the method is defined on a simple resource.
+| Request Builder | header | id | ids | name | setParam | addParam | assocKey | pathKey | paginate | fields | input | inputs | returnEntity |
```

Comment:
It works when I run it locally. Maybe there's something wrong with the spacing, or maybe the metadata for the page is messing it up. Could be that the GitHub viewer just doesn't support it, so it may be fine. Regardless, I'll look into it.